index.html

```diff
@@ -0,0 +1,132 @@
+    href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css"
+    rel="stylesheet"
+    integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD"
+    crossorigin="anonymous"
+  />
+  <link
+    href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css"
+    rel="stylesheet"
+  />
+  <link href="./styles.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Domine&family=Inter:wght@400;700&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Montserrat:wght@100&display=swap" rel="stylesheet"/> 
+    <title>Home Page</title>
+</head>
+<body onload="showFeatures()">
+ 
+   
+        <section class="page-one" >
+            <nav id = 'navbar'>
+            <div class="banner desktop"> 
+                <nav><a href="#"><img src="./images/About/facebook.svg" alt="facebook-logo"></a></nav>
+                  <nav><a href="#"><img src="./images/About/twitter.svg" alt="twitter-logo"></a></nav>
+                  <nav><a href="#">English</a></nav>
+                  <nav><a href="#">My page</a></nav>
+                  <nav><a href="#">Logout</a></nav>
+            </div>
+            <div class="toolbar">
+                <a id="hamburger" class=" hamburger" href="#"><img onclick="popmenu()" alt="Hambruger Icon" src="./images/hamburger.svg"></a>
+                <a class = "logo" href="#"><img alt="Logo Icon" src="./images/logo.svg"></a>
+                <div class="toolbar2 show" id="visible" >
+                    <div><img onclick="popmenu()" src="./images/Home/Icon_Cancel.png" alt="closebtn"></div>
+                    <a class ="toollink" href="./about.html">About</a>
+                    <a class ="toollink" href="#">Program</a>
+                    <a class ="toollink" href="#">Join</a>
+                    <a class ="toollink" href="#">Sponsor</a>
+                    <a class ="toollink" href="#">News</a>
+                    <a class ="toollink" href="#"><img src="./images/About/campaign-logo.svg" alt=""></a>
+                </div>
+            </div>
+        </nav>
+     
+        <section class="home-page">
+            <div class="hero-intro">
+                <img src="./images/Home/hero_intro.svg" alt="intro">
+            </div>
+            <div class="home-section">
+                <p>A joyful celebration believing in the value of openness and sharing, creating a positive change with people from over 80 countries is taking place in October, in Korea.</p>
+            </div>
+            <div class="event-date">
+                <p>2015.10.15(THU) ~ 16(FRI)</p>
+            </div>
+            <div class="venue">
+                <p>@ National Museum of Karea, Art Center Nabi and more</p>
+            </div>
+
+        </section>
+
+        <section class="main-program">
+            <h2>Main Program</h2>
+            <div class="main-section">
+                                <ul class="main-page">
```

Comment:
- [ ] I would suggest you add the divider line under `Main Program` as it is in the original design.
![Capture](https://user-images.githubusercontent.com/63972670/235298555-32afac72-5eea-42dc-9747-05a1042333e6.JPG)